src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPageFairRecoveryOasis.java

```diff
@@ -19,15 +17,23 @@
 
   private static Dimension DESKTOP_SIZE = new Dimension(1920, 768);
 
+  private static final String WIKIA = "arecovery";
+  private static final String WIKIA_ARTICLE = "SyntheticTests/Static_image";
+  private static final String INSTART_LOGIC_SWITCHED_OFF = "InstantGlobals.wgAdDriverInstartLogicRecoveryCountries=[ZZ]";
```

Comment:
If you want to be consistent in your naming: rename `INSTART_LOGIC_SWITCHED_OFF` to `INSTANT_GLOBAL_INSTART_LOGIC_SWITCHED_OFF` or resign from `INSTANT_GLOBAL_*` prefix in other statics, please :)